client/client.go

```diff
@@ -100,6 +105,18 @@ func NewClient(user, password, totpSeed string, options ...Option) (*Client, err
 	return &c, nil
 }
 
+func UseProductionEnvironment(c *Client) {
+	c.BaseURL = BaseURLProduction
+}
+
+func UseStagingEnvironment(c *Client) {
+	c.BaseURL = BaseURLStaging
+}
+
+func UseDevelEnvironment(c *Client) {
+	c.BaseURL = BaseURLDevel
+}
+
```

Comment:
Would prefer to use the option pattern like setting the debug flag.

Reply:
Isn't that exactly what `UseProductionEnvironment`, `UseProductionEnvironment`, and `UseDevelEnvironment` provides?

Reply:
I would go for a 
```go
func WithProductionEnvironment() Option {
	return func(c *Client) {
		c.baseURL = BaseURLProduction
	}
}

func WithStagingEnvironment(c *Client) Option {
	return func(c *Client) {
		c.baseURL = BaseURLStaging
	}
}

func WithDevelEnvironment() Option {
	return func(c *Client) {
		c.baseURL = BaseURLDevel
	}
}
```

Using this pattern you can construct one client using the following line:

```
haricaClient, err := client.NewClient(email, password, seed, client.WithDebug(debug), client.WithDevelEnvironment())
```